docs/README.md

```diff
@@ -50,6 +50,45 @@ version.
 The `website/src/main/resources/docs/README.md` file needs to be updated
 to add the previous versions to a "previous versions" section.
 
+## Preview the website in a forked Github repository
+
+1. Run the following from this directory to build documentation and website modules:
+
+```
+mvn install
```

Comment:
It is a bit controversial without clean, but as maven phases are quite standard these days I believe it is ok.